### ISSUE TITLE
Broken airbrake:deploy task

### ISSUE THREAD

**Issue by olegykz** [airbrake/airbrake#767]
Hey guys,
It looks like that rake task for notifying was broken in 6.2.0
[https://github.com/airbrake/airbrake/blob/master/lib/airbrake/rake/tasks.rb#L33](url)
```ruby
desc 'Notify Airbrake of a new deploy'
task :deploy do
  # how it's possible to get configured airbrake if we didn't load environment?
  unless Airbrake.configured?     
    raise Airbrake::Error, 'airbrake-ruby is not configured'
  end
  
  if defined?(Rails)
    initializer = Rails.root.join('config', 'initializers', 'airbrake.rb')
...
```
**update**
it's possible to preload evironment by **bundle exec rake environment airbrake:deploy**, but airbrake capistrano task doesn't work that way

**Comment by kyrylo** [comment]
Apologies for introducing the bug, and thanks for reporting.

It's a bit hard to write automated tests for that integration, so recent attempts to fix some other issues related to deploys opened this can of worms 😞

**Comment by kyrylo** [comment]
I just released v6.2.1 with the fix: https://github.com/airbrake/airbrake/blob/master/CHANGELOG.md#v621-july-15-2017

Sorry again!
